regression/memory-analyzer/Makefile

```diff
@@ -0,0 +1,21 @@
+default: clean tests.log
+
+clean:
+	find -name '*.exe' -execdir $(RM) '{}' \;
+	find -name '*.out' -execdir $(RM) '{}' \;
+	$(RM) tests.log
+
+test:
```

Comment:
The regression test directory also needs to be added to the parent directory's Makefile and CMakeLists.txt.

Reply:
Done.